src/cassandra/result.rs

```diff
@@ -150,6 +150,24 @@ impl CassResult {
         }
     }
 
+    /// Gets the statement's paging state. This can be used to get the next page of
+    /// data in a multi-page query.
+    ///
+    /// <b>Warning:</b> The paging state should not be exposed to or come from
+    /// untrusted environments. The paging state could be spoofed and potentially
+    // used to gain access to other data.
+    pub fn get_paging_state_token(&self) -> Result<String> {
+        unsafe {
+            let state = CString::new("")?;
+            cass_result_paging_state_token(
```

Comment:
This doesn't look safe to me (it may be OK because `CString` won't allocate for `""`, but it's not pleasant).
Instead, please just accept the pointer and length separately, then build a string from them in the usual way. See https://github.com/Metaswitch/cassandra-rs/blob/master/src/cassandra/value.rs#L412 . You can either do what that does (return a `&str` with the same lifetime as the Result) or copy it to return a `String`.